src/main/scala/no/ndla/conceptapi/model/domain/VisualElement.scala

```diff
@@ -0,0 +1,12 @@
+/*
+ * Part of NDLA concept-api.
+ * Copyright (C) 2019 NDLA
```

Comment:
Dette er vel ei ny fil.
```suggestion
 * Copyright (C) 2020 NDLA
```